Sources/LinuxPlatform/Linux.swift

```diff
@@ -141,5 +141,76 @@ public struct Linux: Platform {
         FileManager.default.temporaryDirectory.appendingPathComponent("swiftly-\(UUID())")
     }
 
+    public func verifySignature(httpClient: SwiftlyHTTPClient, archiveDownloadURL: URL, archive: URL) async throws {
+        guard (try? self.runProgram("gpg", "--version", quiet: true)) != nil else {
+            SwiftlyCore.print("gpg not installed, skipping signature verification.")
+            return
+        }
+
+        let foundKeys = (try? self.runProgram(
+            "gpg",
```

Comment:
I opted to just call through to the `gpg` executable rather than use a library for simplicity, especially since users are going to need to have installed it anyways in order to import the keys.